### PR TITLE
Use checklists

### PR DESCRIPTION
**Pull request by jansepar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -172,6 +172,21 @@ repeating the mistakes of the past.
     - For services, checkout our [best practices](https://mobify.atlassian.net/wiki/display/SYS/Service+Best+Practices).
     - For front-end web apps, checkout our [best practices](https://mobify.atlassian.net/wiki/questions/85917798/what-are-our-best-practices-for-building-front-end-web-application).
 
+### Use checklists
+
+There is a reason NASA makes extensive use of checklists - memories are faulty and unreliable.
+
+Checklists are a great way to ensure:
```
Comment (jeremywiebe) [inline README.md]: I think one of the biggest things that checklists help with is **consistency**.  That's reflected indirectly in the points below, but I think it's worth having it's own bullet point.

File: README.md [hunk 2]
```diff
@@ -172,6 +172,21 @@ repeating the mistakes of the past.
     - For services, checkout our [best practices](https://mobify.atlassian.net/wiki/display/SYS/Service+Best+Practices).
     - For front-end web apps, checkout our [best practices](https://mobify.atlassian.net/wiki/questions/85917798/what-are-our-best-practices-for-building-front-end-web-application).
 
+### Use checklists
```
Comment (johnboxall) [inline README.md]: @jansepar needs Emoji.

File: README.md [hunk 3]
```diff
@@ -172,6 +172,21 @@ repeating the mistakes of the past.
...
+
+There is [plenty developers can learn from pilots](http://lanyrd.com/2015/pycon/sdmfwy/#link-cbgfg) - one is that our memories are faulty and unreliable, leading to inconsistency.
+
+Checklists are a great way to ensure consistency, in areas such as:
+
+- Code changes following best practices
```
Comment (johnboxall) [inline README.md]: @jansepar I feel like its possible to link to direct examples of these being used in the wild.

File: README.md [hunk 4]
```diff
@@ -172,6 +172,21 @@ repeating the mistakes of the past.
...
+- Successful releases
+- Goal planning
+- So much more!
+
+Checklists are also the first step towards automation - the act of writing down a list
```
Comment (johnboxall) [inline README.md]: 💯  though i'd challenge you to write this sentence in 1/2 the words : D

**Comment by ericmuyser** [comment]
Glanced over it, looks good to me!

**Comment by jansepar** [comment]
Gonna assume @johnboxall's +1 on the PR description is a +1!

**Pull request merged**